drivers/docker/config.go

```diff
@@ -253,7 +253,7 @@ var (
 		"ipv6_address":       hclspec.NewAttr("ipv6_address", "string", false),
 		"labels":             hclspec.NewBlockAttrs("labels", "string", false),
 		"load":               hclspec.NewAttr("load", "string", false),
-		"logging": hclspec.NewBlockSet("logging", hclspec.NewObject(map[string]*hclspec.Spec{
+		"logging": hclspec.NewBlock("logging", false, hclspec.NewObject(map[string]*hclspec.Spec{
 			"type":   hclspec.NewAttr("type", "string", false),
```

Comment:
Also, does this affect json job parsing that use these attributes?

Reply:
Not sure, but this is currently broken - any use of logging stanza gets:

```
driver start failed: failed to decode driver config: [pos 308]: readContainerLen: Unrecognized descriptor byte: hex: 82, decimal: 130
```

It sounds like we should follow up with json serialization and compatibility testing as a separate work.